tox.ini

```diff
@@ -37,6 +37,21 @@ environment =
     MYSQL_ALLOW_EMPTY_PASSWORD=1
     MYSQL_USER=quay
 
+[testenv:py39-unit]
+setenv =
+    PYTHONDONTWRITEBYTECODE = 1
+    PYTHONPATH={toxinidir}{:}{toxinidir}
+    TEST=true
+    SKIP_DB_SCHEMA=true
+    MARKERS="not e2e"
+    TEST_DATABASE_URI=sqlite:///data/quay_sqlite_test.db
+    registry: FILE=test/registry/registry_tests.py
+    e2e: MARKERS="e2e"
```

Comment:
These markers make sense only in the generic [testenv], i.e. these envvars will be set in py39-registry and py39-e2e respectively. They shouldn't be in `[testenv:py39-unit]`, as they don't change anything here.